spec.md

```diff
@@ -498,10 +531,6 @@ message VolumeInfo {
   // NFS share). If set, it MUST be non-zero.
   uint64 capacity_bytes = 1;
 
```

Comment:
I guess the use case I am trying to understand is this:
1. `CreateVolume(VolumeCapability...)` where `VolumeCapability` contains `access_mode` set to `SINGLE_NODE_READER_ONLY`.
2. The volume plugin satisfies the request by provisioning a volume that is `MULTI_NODE_READER_ONLY`.
3. The `CreateVolumeResponse` does not contain the actual `access_mode`, because it was removed from `VolumeInfo`.

How would the CO figure out the actual `access_mode` of the newly created volume? By issuing a follow up `ValidateVolumeCapabilities` call?